azurelinuxagent/ga/exthandlers.py

```diff
@@ -310,11 +309,68 @@ def handle_ext_handlers(self, etag=None):
                 logger.info("Extension handling is on hold")
                 return
 
+        dep_level = None
+        deps_wait_until = None
+        handlers_wait_until = datetime.datetime.utcnow()
+        prev_handler = None
+
         self.ext_handlers.extHandlers.sort(key=operator.methodcaller('sort_key'))
         for ext_handler in self.ext_handlers.extHandlers:
-            # TODO: handle install in sequence, enable in parallel
+            # Use updated time limit for each dependency level
+            if dep_level != ext_handler.sort_key():
+                dep_level = ext_handler.sort_key()
+                deps_wait_until = handlers_wait_until
+                handlers_wait_until += datetime.timedelta(minutes=DEFAULT_EXT_TIMEOUT_MINUTES)
+
+            # Make sure that the previous extension is installed.
```

Comment:
minor: update the comment to mention install/enable instead of only install

Reply:
Install and enable were already run for the extension. This comment should say "Make sure that the previous extension reported status successfully" or something like that.